samples/sampler/stories/qa/FixedPopupPanels.js

```diff
@@ -314,7 +314,7 @@ export const WithVariousItems = () => {
 							<Button size="small" disabled onClick={nextPanel} onKeyDown={handleKeyDown}>Button1</Button>
 							<br />
 							<br />
-							<Button size="small">Button2</Button>
+							<Button size="small" onClick={handleClose}>Button2</Button>
```

Comment:
How about change label to Close
```suggestion
							<Button size="small" onClick={handleClose}>Close</Button>
```